libs/model/account.hpp

```diff
@@ -25,22 +26,59 @@ namespace iroha {
      * Account Model
      */
     struct Account {
+
+      struct Permissions {
+
+        /**
+         * Can account add assets to own account;
+         * Dangerous operation - require high number of quorum;
+         */
+        bool issue_assets;
+
+        /**
+         * Can account create new type of assets;
+         * Dangerous operation - require high number of quorum;
+         */
+        bool create_assets;
+
+        /**
+         * Can account read private information of other accounts;
+         * Auditor's permission
+         */
+        bool read_all_accounts;
+
+        /**
+         * Available change only read_all_accounts and unlimited_transfer
+         * Administrator's account permission
+         */
+        bool set_permissions;
+
+        /**
+         * Can account provide unlimited transfers to other accounts;
+         * User's account permission
+         */
+        bool unlimited_transfer;
+      };
+
       /**
-       * ACTIVE - active account
-       * SUSPENDED - not active account
-       *
+       * Is this account banned for operations
        */
-      enum State { ACTIVE, SUSPENDED };
+      bool is_banned;
 
-      /*
-       * Account state
+      /**
+       * Account permissions
        */
-      State state;
+      Permissions permissions;
```

Comment:
Why account has permission structure?
I want to know your reason or your proposal about data structure.

I think we only share jira's information about data strcture, don't we?